src/main/java/com/enterprise/redcord/RedCordController.java

```diff
@@ -47,14 +47,14 @@ public String index(@ModelAttribute("topicEntry") String topicName, Model model)
             topicId = topicService.fetchByTopicName(topicName);
             messages = messageService.fetchByTopicId(topicId);
             topics = topicService.fetchAllTopics();
+            model.addAttribute("messages", messages);
+            model.addAttribute("topics", topics);
         } catch (ExecutionException e) {
             e.printStackTrace();
         } catch (InterruptedException e) {
             e.printStackTrace();
         }
         logger.trace("Accessed index method in RedCordController.");
-        model.addAttribute("messages", messages);
-        model.addAttribute("topics", topics);
```

Comment:
I'd keep this where it was.  You're assuming an exception was not thrown.  If you're beyond the catch blocks, there's a chance that these were not properly populated.